Mac/BuildScript/build-installer.py

```diff
@@ -385,6 +385,14 @@ def library_recipes():
 
     return result
 
+def compilerCanOptimize():
+    """
+    Return True iff the default Xcode version can use PGO
+    """
+    # The version check is pretty conservative, can be
+    # adjusted after testing
```

Comment:
Maybe add "bpo-42235: " prefix to give more context to this choice.

Reply:
Agreed.

Reply:
Perfect, thanks ;-)